kieserver/modules/kieserver/module.yaml

```diff
@@ -36,6 +36,9 @@ artifacts:
 - name: "slf4j-simple.jar"
   # slf4j-simple-1.7.22.redhat-2.jar
   md5: "62cc6eeb72e2738e3acc8957ca95f37b"
+- name: "kie-server-services-jbpm-cluster-7.52.0.Final-redhat-00004.jar"
+  md5: "44f302877d28e2493fc28ead4dfb6dea"
+  dest: /opt/deps
```

Comment:
I would prefer to use more specific naming.
what about 
``` /opt/kie/dependencies/jbpm-clustering ```

?

Reply:
@spolti  Why so long ? Isn't a repo, /opt/jbpm-clustering ?

Reply:
consistency.

Reply:
Ok, applied